js-peer/src/lib/libp2p.ts

```diff
@@ -67,24 +68,24 @@ export async function startLibp2p() {
     ],
     services: {
       pubsub: gossipsub({
-        allowPublishToZeroPeers: true,
+        allowPublishToZeroTopicPeers: true,
         msgIdFn: msgIdFnStrictNoSign,
         ignoreDuplicatePublishError: true,
       }),
       dht: kadDHT({
-        protocolPrefix: "/universal-connectivity",
+        protocol: "/universal-connectivity",
```

Comment:
If we want to maintain compatibility with existing universal connectivity peers:

```suggestion
        protocol: "/universal-connectivity/kad/1.0.0",
```

Reply:
thanks for the review - done